superset-frontend/src/pages/ThemeList/index.tsx

```diff
@@ -215,6 +233,96 @@ function ThemesList({
     addSuccessToast(t('Theme imported'));
   };
 
+  const handleSetSystemDefault = (theme: ThemeObject) => {
+    Modal.confirm({
+      title: t('Set System Default Theme'),
+      content: t(
+        'Are you sure you want to set "%s" as the system default theme? This will apply to all users who haven\'t set a personal preference.',
```

Comment:
What controls are we giving to the admins to decide whether individual users can set their own themes? I am thinking that some orgs might just want everybody to stick to the org theme.

Reply:
nothing yet ... but users with CRUD write access can set temp "local" theme